internal/controller/controller.go

```diff
@@ -22,6 +22,8 @@ import (
 
 const MiB = 1 << 20
 
+const MaxTailCalls = 29
```

Comment:
Making all users define a "sensible" `MaxTailCalls` sounds a bit tedious. How about remove this here and move it to `tracer` to be used as default if the `tracer.Config.MaxTailCalls` is zero?